jobs/pulse3d/pulse3d-worker/src/main.py

```diff
@@ -112,7 +124,12 @@ async def process(con, item):
             try:
                 if not re_analysis:
                     logger.info("Writing time force data to parquet file for new upload")
-                    time_force_df = recordings[0].to_dataframe(tmpdir)
+                    if PULSE3D_VERSION == "0.24.6":
```

Comment:
Can you add a TODO note here to use the semver package to compare if `PULSE3D_VERSION < "0.24.9"` instead